File: java/src/main/java/_26/RemoveDuplicatesfromSortedArray.c
int removeDuplicates(int* nums, int numsSize) {
    if (numsSize == 0)
        return numsSize;
    int i = 0;
    int j = i + 1;
    while (j < numsSize)
    {
        if (nums[i] != nums[j])
        {
            nums[++ i] = nums[j];
        }
        j++;
    }
    return i + 1;
}

File: java/src/main/java/_26/RemoveDuplicatesfromSortedArray.java
package _26;

public class RemoveDuplicatesfromSortedArray {
    public int removeDuplicates(int[] nums) {
        if (nums.length < 1) {
            return nums.length;
        }
        int l = 0, r = 1;
        while (r < nums.length) {
            if (nums[r] != nums[l]) {
                nums[++ l] = nums[r];
            }
            r ++;
        }
        return l + 1;    
    }    
}
